site/en/_data/nav.yml

```
@@ -10,6 +10,8 @@
       url: web
     - title: Games
       url: games
+    - title: Building for Education
```

```suggestion
    - title: Education
```

This top nav is going to change so we want to go with just "Education" here.